identifiers/country-tr.csv

```diff
@@ -1,2 +1,93 @@
 id,name
 ocd-division/country:tr,Turkey
+ocd-division/country:tr/province:01,Adana
+ocd-division/country:tr/province:02,Adıyaman
+ocd-division/country:tr/province:03,Afyonkarahisar
+ocd-division/country:tr/province:04,Ağrı
+ocd-division/country:tr/province:05,Amasya
+ocd-division/country:tr/province:06,Ankara
+ocd-division/country:tr/province:06/ed:1,Ankara I
+ocd-division/country:tr/province:06/ed:2,Ankara II
+ocd-division/country:tr/province:06/ed:3,Ankara III
+ocd-division/country:tr/province:07,Antalya
+ocd-division/country:tr/province:08,Artvin
+ocd-division/country:tr/province:09,Aydın
+ocd-division/country:tr/province:10,Balıkesir
+ocd-division/country:tr/province:11,Bilecik
+ocd-division/country:tr/province:12,Bingöl
+ocd-division/country:tr/province:13,Bitlis
+ocd-division/country:tr/province:14,Bolu
+ocd-division/country:tr/province:15,Burdur
+ocd-division/country:tr/province:16,Bursa
+ocd-division/country:tr/province:16/ed:1,Bursa I
+ocd-division/country:tr/province:16/ed:2,Bursa II

```

Comment:
I would prefer to switch everything to "ed" if needed. But, I find it confusing, since in multiple official website, the electoral districts are linked to their province. 

@jloutsenhizer @jpmckinney any thoughts ?

Reply:
+1 to using `ed` for all of the electoral districts. This also simplifies things if a province adds or removes enough seats to change the number of electoral districts as well.

Reply:
I updated the type to "ed" for all the electoral districts. However, I used the ISO code of province for simplicity.